pkg/abstractions/image/build.go

```diff
@@ -37,6 +40,11 @@ type Builder struct {
 	tailscale     *network.Tailscale
 }
 
+type ContainerCommand struct {
```

Comment:
If we end up supporting things like `with_cuda_version` or something similar (things that aren't necessarily a container command), is this the right naming? I was thinking maybe something along the lines of `ContainerBuildStep` (or something like that)

Users would need to know either way, but for internal clarity

Reply:
Yeah good point. I'll change it.

Reply:
Updated to `BuildStep`.